mlir/test/mlir-tblgen/gen-dialect-doc.td

```diff
@@ -85,7 +93,19 @@ def TestEnum :
 // CHECK: [TOC]
 
 // CHECK-NOT: [TOC]
+
 // CHECK: test.e
+// CHECK: _Op with a summary_
+// CHECK: Op with a description
+// CHECK: Operands:
+// CHECK: | Operand | Description |
+// CHECK: | :-----: | ----------- |
+// CHECK: | `tensor` | |
+// CHECK: Results:
+// CHECK: | Result | Description |
+// CHECK: | :----: | ----------- |
+// CHECK: | `output` | |
```

Comment:
If you care about the new lines, you should use CHECK-NEXT to ensure that there are no other line in-between here.

Reply:
Will do! Is there any way to match an empty line? We also care about the presence of a newline in some places

Reply:
I think `// CHECK: {{^$}}` may do?

Reply:
Ah, found `CHECK-EMPTY:` 

https://llvm.org/docs/CommandGuide/FileCheck.html#the-check-empty-directive

Reply:
Done, added CHECK-NEXT and CHECK-EMPTY where it made sense.

Looks like CHECK-EMPTY doesn't have a `NEXT` form (tried CHECK-NEXT-EMPTY / CHECK-EMPTY-NEXT), but I think enforcing this way is probably fine given that the bug is that we just need to ensure there is _some_ empty line(s) between doc blocks.